tests/unit/test_node_execution_time.py

```
@@ -0,0 +1,208 @@
+        """Test that GenVMHost.get_contract_schema() measures execution time"""
+        genvm_host = GenVMHost()
+        contract_code = b"def get_schema(): return 'test schema'"
+        mock_execution_time = 0.2  # 200ms
+
+        with patch("backend.node.genvm.base._run_genvm_host") as mock_run:
+            mock_execution_result = create_mock_execution_result()
+
+            with patch("time.time", side_effect=[2000.0, 2000.0 + mock_execution_time]):
+                mock_run.return_value = mock_execution_result
+                result = await genvm_host.get_contract_schema(contract_code)
+                assert result.processing_time == int(
```

Ruff (0.12.2)

Use of `assert` detected

(S101)